tests/robot-tests/tests/admin_and_public_2/bau/publish_methodology_publication_update.robot

```diff
@@ -131,11 +131,13 @@ Update methodology details
     user waits until page contains element    xpath://h2[contains(text(),'Edit methodology summary')]
     user clicks radio    Set an alternative title
     user enters text into element    id:updateMethodologyForm-title    ${PUBLICATION_NAME}-methodology update
+    user waits until button is clickable    Update methodology
```

Comment:
You might already be aware, but this does have a little bit of overlap with `user clicks button`, which in turn calls `user clicks element` which also checks the element is enabled. But I'm inclined to says that's fine - `user waits until button is clickable` is doing other checks besides checking the element is enabled that may prevent a test failure.

Reply:
Yeah I am aware of that , rather than interested to put some sleep, I just thought to do this way for the sake of time killing . The actual problem is DOM is not getting updated as fast as the robot framework would require due to which its failing. Ideally I would be more interested in using some method like waitforFunction which exist in playwright - helps in preventing racing conditions.